Apps/APN/ios/AppDelegate.mm

```diff
@@ -76,7 +77,9 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   self.window.rootViewController = rootViewController;
   [self.window makeKeyAndVisible];
   
-  [pnHandlerObj setupCustomerIOClickHandling:self];
+  [pnHandlerObj setupCustomerIOClickHandling];
+
+  [RNNotifications startMonitorNotifications];
```

Comment:
I think it would be valuable to add a comment here with reference to the link.

Reply:
Are you suggesting adding a link to the docs for `react-native-notifications`? 

If we did add a link, that makes it feel like our sample apps are a replacement to the `react-native-notifications` documentation. I currently do not think that's a good idea. 

Customers should be referring to the official docs of 3rd party SDKs to learn how to install them. 

Based on a suggestion from Rehan in the Flutter app, I [did add a comment suggesting that customers review the documentation for 3rd party SDKs](https://github.com/customerio/customerio-flutter/pull/105/commits/ec9142de5e2ff12845d5593d59babe57de2af1af). Is that what you're thinking?

Reply:
Yes, I believe a similar comment should work for my suggestion as well.